integration-libs/cdc/root/service/cdc-js.service.ts

```diff
@@ -36,6 +36,8 @@ import { catchError, filter, switchMap, take, tap } from 'rxjs/operators';
 import { CdcConfig } from '../config/cdc-config';
 import { CdcAuthFacade } from '../facade/cdc-auth.facade';
 
+const defaultSessionTimeOut = 3600;
+const setAccountInfoAPI = 'accounts.setAccountInfo';
```

Comment:
For easier extensibility for customers, it would be good to move those conts to become `protected readonly` properties of the class.

Note: fixing this can be moved after 6.0

Reply:
Indeed, we do have the session expiration defined in the `cdc-feature.module.ts` file. However, this is just a backup in case customers deliberately leave it empty. Also, the API names are not meant to be changed / configured by the customer, since changing them would cause the entire native UI integration to fail. So, I would not recommend moving API names to `cdc-feature.module.ts`.


Please let me know your thoughts @Platonn

Reply:
Thanks for you explanation. Fair enough, let's leave them as consts.
- expiration is already eaisly customizable via Spartacus CDC config 👍 
- API name is also not a subject of customization, so also can stay as consts.

Reply:
thanks @Platonn 👍